FeedAPIChallenge/Feed API/FeedImageMapper.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Sergio Khalil Bello Garcia on 30/08/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+final class FeedImageMapper {
+	private struct Root: Decodable {
+		let items: [Image]
+
+		var feed: [FeedImage] {
+			return items.map { $0.feedImage }
```

Comment:
Suggestion: another way of using the `map` function is with key paths:
```suggestion
			return items.map(\.feedImage)
```